### ISSUE TITLE
Release (2.0.1) not working with J8

### ISSUE THREAD

**Issue by H3llK33p3r** [fonimus/ssh-shell-spring-boot#237]
Hi an issue on sshd core has been introduced with 2.9.0 causing a no such method error for Java 8. Version 2.9.1 should fix the issue. 

https://github.com/apache/mina-sshd/blob/master/docs/changes/2.9.1.md 

Thank you in advance to fix the dependencies in your project. 
Moreove are you aware that new spring shell enables by default an interactive front bocking JVM using spring MVC ? May be you should inform the uses to disable this feature using spring mvc  (`spring.shell.interactive.enabled=false`)?

**Comment by H3llK33p3r** [comment]
Already fixed by the bot : https://github.com/fonimus/ssh-shell-spring-boot/pull/235

**Comment by fonimus** [comment]
Hi, thanks for creating this issue.

I just released next version 2.0.2 with patch 2.9.1 of sshd core. It should be available in the next hours.

I added a warning in the readme because the new spring shell version enables the interactive shell by default, so the version 2.0.0 of this project does too.

But for me the interactive shell does not block JVM ; I added controllers in samples to tests it easily (maybe it does block on some platform, or some terminal)
